best-practice/definitions.ts

```diff
@@ -9,6 +9,56 @@ const github: readonly IBestPractice[] = [
     howToCheck: "Manual. View the repository on https://github.com",
     howToExempt: "N/A",
   },
+  {
+    name: "Branch Protection",
+    owner: "[@guardian](https://github.com/orgs/guardian/teams/all)",
+    description:
+      "Enable branch protection for the default branch, ensuring changes are reviewed before being deployed.",
+    howToCheck: "Manual. View the repository on https://github.com",
+    howToExempt: "N/A",
+  },
+  {
+    name: "Access",
+    owner: "[@guardian](https://github.com/orgs/guardian/teams/all)",
+    description:
+      "Grant access on a team basis, rather than directly to individuals.",
+    howToCheck: "Manual. View the repository on https://github.com",
+    howToExempt: "N/A",
+  },
+  {
+    name: "Collaborators",
+    owner: "[@guardian](https://github.com/orgs/guardian/teams/all)",
+    description:
+      "Include at least one GitHub team as a collaborator with admin access.",
+    howToCheck: "Manual. View the repository on https://github.com",
+    howToExempt: "N/A",
+  },
+  {
+    name: "Archiving",
+    owner:
+      "[DevX Operations](https://github.com/orgs/guardian/teams/devx-operations)",
+    description: "Repositories that are no longer used should be archived.",
+    howToCheck: "Manual. View the repository on https://github.com",
+    howToExempt: "N/A",
+  },
+  {
+    name: "Topics",
+    owner:
+      "[DevX Security](https://github.com/orgs/guardian/teams/devx-security)",
+    description:
+      "Repositories should have a topic, to help understand what is in production.",
+    howToCheck: "Manual. View the repository on https://github.com",
+    howToExempt: "N/A",
```

Comment:
repos owned **only** by non-p&e teams are exempt

ie
interactives -> exempt
devx & interactives -> not exempt
none -> not exempt